src/main/java/hometask3/Vector.java

```diff
@@ -0,0 +1,39 @@
+package hometask3;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class Vector {
+    private int x;
+    private int y;
+    private int z;
+
+    public Vector(int x, int y, int z) {
```

Comment:
You can also create constructor without parameters that will create vector with random coordinates (e.g. you can implement method getRandomCoordinate() and set this.x = getRandomCoordinate();... inside of constructor)